docker/dockerfile

```diff
@@ -0,0 +1,78 @@
+FROM ubuntu:18.04
+
+MAINTAINER Sai Raghav Keesara <sfk5555@psu.edu>
+
+RUN apt-get -y update && apt-get -y upgrade
+RUN apt-get -y install vim python-pip git wget unzip yum software-properties-common default-libmysqlclient-dev
+RUN echo "deb https://dl.bintray.com/sbt/debian /" | tee -a /etc/apt/sources.list.d/sbt.list
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 642AC823
+RUN apt-get update
+RUN apt-get install sbt
+RUN pip install --upgrade pip
+
+# --------------------------------------------- Python 3 -------------------------------------------
+# TODO: uncomment below code when pdfmef is ported to python 3
+
+#RUN apt-get -y install vim python-pip python3 python3-pip git wget unzip yum software-properties-common
+#RUN pip3 install --upgrade pip
```

Comment:
current pdfmef doesn't work with python3

Reply:
Yes, I have commented on the code which installs Python 3 and other related dependencies. The current dockerfile  is meant for Python 2 only. I have retained the commented code so so that we can uncomment that once we made pdfmef fully-compatible with python 3 in the future. Let me know if you want to delete the commented code instead.